assets/component-image-with-text.css

```diff
@@ -40,11 +37,18 @@
 }
 
 .image-with-text__media--placeholder {
-  background-color: rgba(var(--color-foreground), 0.04);
   position: relative;
   overflow: hidden;
 }
 
+.image-with-text__media--placeholder:after {
+  content: '';
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background: rgba(var(--color-foreground), 0.04);
```

Comment:
Well image-with-text already had that. I just had to move it to a pseudo element to play nice with my changes. But if other section placeholders don't have the same slight darkening effect maybe it should be removed. @melissaperreault thoughts on that?

Reply:
If it doesn't affect merchants assets this is low priority and can be investigated later if it is not an easy fix. This is palceholder styling, it's great if they share the same outcome but not concerned to ship this PR without.

Reply:
I remember why it needs to be in a pseudo element. Because including it on the `image-with-text__media` overrides the background color it inherits from the `gradient` class, which leads to a 4% transparent background, as opposed to a solid background with a 4% overlay on top. The former can cause this sort of side effect where shadows can show through the bottom of the placeholder
<img src="https://screenshot.click/14-22-ot2fv-sqxms.png" width="400"/>